mne/viz/_3d.py

```diff
@@ -1050,7 +1050,7 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
 
     # initialize figure
     if fig is None:
-        fig = mlab.figure(bgcolor=(0.0, 0.0, 0.0), size=(800, 800))
+        fig = mlab.figure(bgcolor=(0.5, 0.5, 0.5), size=(800, 800))
```

Comment:
@larsoner @christianbrodbeck @dengemann I changed the background color for the 3d plot alignment. Looks like this now. I prefer the contrast now as we can better see the helmet.

<img width="590" alt="screen shot 2018-02-01 at 10 09 33" src="https://user-images.githubusercontent.com/161052/35670180-0709f2ec-0738-11e8-9bc4-eed53b57261f.png">